.storybook/formDecorator.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import { Button } from '@dhis2/ui-core'
+
+import { Form, FormSpy } from '../src'
+
+const formProps = {
+    onSubmit: values => {
+        console.log(
+            '++++++++++++++++\n',
+            'Form was submitted with values:\n',
+            values,
+            '\n----------------'
+        )
+    },
+    mutators: {},
+}
+
+class FormWithSpyAndSubmit extends React.Component {
```

Comment:
Hendrik and I talked a bit about using this component as a decorator or not. I was suggesting waiting with adding abstractions for the tests. Just because it avoids adding an abstraction too soon that might not match all usecases and potentially adding an extra layer to comprehend, etc.

Seeing this in action, it seems fine to me. The tests seem quite readable, and if it turns out that we've locked ourselves into an abstraction that doesn't work for all usecases, I think it shouldn't be too hard to move to a different approach.